plugin.rb

```diff
@@ -94,13 +94,20 @@ def create_post
       perform_fetch do
         @post = Babble::PostCreator.create(current_user, post_creator_params)
         if @post.persisted?
+          success_callback
           respond_with @post, serializer: Babble::PostSerializer
         else
           respond_with_unprocessable
         end
       end
     end
 
+    def success_callback
+      if (SiteSetting.babble_remote_post)
+        response = RestClient.post(SiteSetting.babble_remote_url, { current_user: current_user.username, message: @post.cooked }) unless params[:from_discord]
```

Comment:
Can we change the params to `params[:skip_callback]`, so that there's not discord-specific language in there? Also, you won't need to assign the response to a variable, since we don't use it

```
RestClient.post(SiteSetting.babble_remote_url, { current_user: current_user.username, message: @post.cooked }) unless params[:skip_callback]
```